src/services/TestNumberService.ts

```diff
@@ -0,0 +1,130 @@
+import {AWSError} from "aws-sdk";
+import {TestNumber} from "../models/TestNumber";
+import {Service} from "../models/injector/ServiceDecorator";
+import {HTTPResponse} from "../utils/HTTPResponse";
+import {DynamoDBService} from "./DynamoDBService";
+
+@Service()
+export class TestNumberService {
+    public readonly dbClient: DynamoDBService;
+
+    /**
+     * Constructor for the TestNumberService class
+     * @param dynamo
+     */
+    constructor(dynamo: DynamoDBService) {
+        this.dbClient = dynamo;
+    }
+
+    /**
+     * Creates a new test number in the database.
+     * @param activity - the payload containing the activity
+     */
+    public async createTestNumber(): Promise<TestNumber> {
+        return this.getLastTestNumber()
+            .then(lastTestNumber => {
+                const testNumber: TestNumber = this.createNextTestNumberObject(lastTestNumber)
+                return this.dbClient.put(testNumber)
+                    .then(() => {
+                        return testNumber;
+                    })
+                    .catch((error: AWSError) => {
+                        throw new HTTPResponse(error.statusCode, { error: `${error.code}: ${error.message}
+                        At: ${error.hostname} - ${error.region}
+                        Request id: ${error.requestId}` });
+                    });
+            })
+    }
+
+    public async getLastTestNumber(): Promise<TestNumber>{
+        return this.dbClient.scan()
+            .then((data:any) => {
+                if(data.Count === 0){
+                    return {testNumber: 'W01A000',
```

Comment:
Would be good to have the initial start number configurable, so that in production we can start with a different number once start range is confirmed
so may be instead of hardcoding W01A000, read it from environment config pls

Reply:
done